src/08-accessing.js

```diff
@@ -87,7 +97,9 @@ function getCountOfPhoneNumbers(person) {}
  *  getFirstPhoneNumber(person);
  *  //> 7185550921
  */
-function getFirstPhoneNumber(person) {}
+function getFirstPhoneNumber(person) {
+  return person.numbers.shift()
```

Comment:
Be careful with ` .shift() ` this actually changes the array permanently.  How else can we index the firt array item?